spec/support/schema.rb

```diff
@@ -171,6 +172,7 @@ def self.create
         t.string   :title
         t.text     :subject_header
         t.text     :body
+        t.boolean  :published, default: true
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.